SlowCheetah.JDT/Processors/JdtRemove.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Represents a recursive JDT transformation
+    /// </summary>
+    internal class JdtRemove : JdtProcessor
+    {
+        private const string PathAttribute = "path";
+
+        private bool removedThisNode;
+
+        /// <inheritdoc/>
+        public override string Verb { get; } = "remove";
+
+        /// <inheritdoc/>
+        public override void Process(JObject source, JObject transform)
+        {
+            this.removedThisNode = false;
+
+            JToken removeValue;
+            if (transform.TryGetValue(JsonUtilities.JdtSyntaxPrefix + this.Verb, out removeValue))
+            {
+                this.Remove(source, removeValue, true);
```

Comment:
`this.Remove` should return true/false indicating to continue or not. Then this can change to `if (!this.Remove(source, removeValue, true) { return; }`. Then the `this.Successor.Process` does not need to be conditional below.